rainforest/rainforest.go

```diff
@@ -57,7 +57,12 @@ func NewClient(token string, debug bool) *Client {
 		baseURL, _ = url.Parse(currentBaseURL)
 	}
 
-	return &Client{client: http.DefaultClient, BaseURL: baseURL, ClientToken: token, LastResponseHeaders: http.Header{}, DebugFlag: debug}
+	return &Client{client: http.DefaultClient, BaseURL: baseURL, clientToken: token, LastResponseHeaders: http.Header{}, DebugFlag: debug}
+}
+
+// ClientToken returns the API authentication token for the client.
+func (c *Client) ClientToken() string {
```

Comment:
Ah. Setting the field value in the test manually has issues with race conditions?